TriblerGUI/test/widgets/trustpage/js/test_drawing.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * This file tests the drawing.js file with unit tests
+ */
+assert = require("assert");
+drawing = require("../../../../widgets/trustpage/js/drawing.js");
+
+describe("drawing.js", function () {
+    describe("getStrokeWidth", function () {
```

Comment:
You assigned yourself as a reviewer last-minute, that is ok, but please do a little effort in reviewing and read the code under test.